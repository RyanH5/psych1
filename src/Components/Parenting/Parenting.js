import React from 'react';

export const Parenting = () => {
  return (
    <div>
      <h1>Seminars for Parents</h1>
      <p>
      I have been providing seminars to parents and families for over 25 years. Parents often say, "Our kids didn't come with directions." As a parent of 4 children and as a psychologist, I appreciate the challenges parents face in raising their children as healthy, happy and competent individuals. The goal of the seminars is to provide parents "directions" in understanding why their child is struggling and how they can help their child develop his/her body, brain, and emotions.
      </p>
      <p>
      One of the most important skills children need to learn is what they can do when things become difficult in their life. Children can learn maladaptive coping strategies that can lead to disruption in the home, school and with friends. The focus of the workshops is to provide parents with resources to help their children develop skills in managing challenges in the different parts of their life. My approach is to provide practical and easy to understand strategies that can use on a daily basis.
      </p>
      <h2>Effective Parenting Seminars</h2>
      <p>
      These are intensive parenting seminars that are offered several times a year. Each seminar is four-weeks. The seminar focuses on assisting parents in developing additional parenting skills. Topics include: school related issues, sibling issues, limit setting and emotional regulation. Specific plans are developed and time is given for parents to implement them. Parents will be able to discuss each week the effectiveness of the plans. The seminars are very practical and skill-based.
      </p>
      <h2>Upcoming Seminars</h2>
      <p>
        No upcoming seminars for parents.	Please check back later.
      </p>

    </div>
  )
}