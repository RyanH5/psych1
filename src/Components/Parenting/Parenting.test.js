import React from 'react';
import { shallow } from 'enzyme';
import { Parenting }from './Parenting';

describe('Parenting', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<Parenting />)

    expect(wrapper).toMatchSnapshot();
  });
});