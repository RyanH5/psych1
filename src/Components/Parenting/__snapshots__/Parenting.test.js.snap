// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parenting should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Seminars for Parents
</h1>,
        <p>
          I have been providing seminars to parents and families for over 25 years. Parents often say, "Our kids didn't come with directions." As a parent of 4 children and as a psychologist, I appreciate the challenges parents face in raising their children as healthy, happy and competent individuals. The goal of the seminars is to provide parents "directions" in understanding why their child is struggling and how they can help their child develop his/her body, brain, and emotions.
</p>,
        <p>
          One of the most important skills children need to learn is what they can do when things become difficult in their life. Children can learn maladaptive coping strategies that can lead to disruption in the home, school and with friends. The focus of the workshops is to provide parents with resources to help their children develop skills in managing challenges in the different parts of their life. My approach is to provide practical and easy to understand strategies that can use on a daily basis.
</p>,
        <h2>
          Effective Parenting Seminars
</h2>,
        <p>
          These are intensive parenting seminars that are offered several times a year. Each seminar is four-weeks. The seminar focuses on assisting parents in developing additional parenting skills. Topics include: school related issues, sibling issues, limit setting and emotional regulation. Specific plans are developed and time is given for parents to implement them. Parents will be able to discuss each week the effectiveness of the plans. The seminars are very practical and skill-based.
</p>,
        <h2>
          Upcoming Seminars
</h2>,
        <p>
          No upcoming seminars for parents. Please check back later.
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Seminars for Parents",
        },
        "ref": null,
        "rendered": "Seminars for Parents",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "I have been providing seminars to parents and families for over 25 years. Parents often say, \\"Our kids didn't come with directions.\\" As a parent of 4 children and as a psychologist, I appreciate the challenges parents face in raising their children as healthy, happy and competent individuals. The goal of the seminars is to provide parents \\"directions\\" in understanding why their child is struggling and how they can help their child develop his/her body, brain, and emotions.",
        },
        "ref": null,
        "rendered": "I have been providing seminars to parents and families for over 25 years. Parents often say, \\"Our kids didn't come with directions.\\" As a parent of 4 children and as a psychologist, I appreciate the challenges parents face in raising their children as healthy, happy and competent individuals. The goal of the seminars is to provide parents \\"directions\\" in understanding why their child is struggling and how they can help their child develop his/her body, brain, and emotions.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "One of the most important skills children need to learn is what they can do when things become difficult in their life. Children can learn maladaptive coping strategies that can lead to disruption in the home, school and with friends. The focus of the workshops is to provide parents with resources to help their children develop skills in managing challenges in the different parts of their life. My approach is to provide practical and easy to understand strategies that can use on a daily basis.",
        },
        "ref": null,
        "rendered": "One of the most important skills children need to learn is what they can do when things become difficult in their life. Children can learn maladaptive coping strategies that can lead to disruption in the home, school and with friends. The focus of the workshops is to provide parents with resources to help their children develop skills in managing challenges in the different parts of their life. My approach is to provide practical and easy to understand strategies that can use on a daily basis.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Effective Parenting Seminars",
        },
        "ref": null,
        "rendered": "Effective Parenting Seminars",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "These are intensive parenting seminars that are offered several times a year. Each seminar is four-weeks. The seminar focuses on assisting parents in developing additional parenting skills. Topics include: school related issues, sibling issues, limit setting and emotional regulation. Specific plans are developed and time is given for parents to implement them. Parents will be able to discuss each week the effectiveness of the plans. The seminars are very practical and skill-based.",
        },
        "ref": null,
        "rendered": "These are intensive parenting seminars that are offered several times a year. Each seminar is four-weeks. The seminar focuses on assisting parents in developing additional parenting skills. Topics include: school related issues, sibling issues, limit setting and emotional regulation. Specific plans are developed and time is given for parents to implement them. Parents will be able to discuss each week the effectiveness of the plans. The seminars are very practical and skill-based.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Upcoming Seminars",
        },
        "ref": null,
        "rendered": "Upcoming Seminars",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "No upcoming seminars for parents. Please check back later.",
        },
        "ref": null,
        "rendered": "No upcoming seminars for parents. Please check back later.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Seminars for Parents
</h1>,
          <p>
            I have been providing seminars to parents and families for over 25 years. Parents often say, "Our kids didn't come with directions." As a parent of 4 children and as a psychologist, I appreciate the challenges parents face in raising their children as healthy, happy and competent individuals. The goal of the seminars is to provide parents "directions" in understanding why their child is struggling and how they can help their child develop his/her body, brain, and emotions.
</p>,
          <p>
            One of the most important skills children need to learn is what they can do when things become difficult in their life. Children can learn maladaptive coping strategies that can lead to disruption in the home, school and with friends. The focus of the workshops is to provide parents with resources to help their children develop skills in managing challenges in the different parts of their life. My approach is to provide practical and easy to understand strategies that can use on a daily basis.
</p>,
          <h2>
            Effective Parenting Seminars
</h2>,
          <p>
            These are intensive parenting seminars that are offered several times a year. Each seminar is four-weeks. The seminar focuses on assisting parents in developing additional parenting skills. Topics include: school related issues, sibling issues, limit setting and emotional regulation. Specific plans are developed and time is given for parents to implement them. Parents will be able to discuss each week the effectiveness of the plans. The seminars are very practical and skill-based.
</p>,
          <h2>
            Upcoming Seminars
</h2>,
          <p>
            No upcoming seminars for parents. Please check back later.
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Seminars for Parents",
          },
          "ref": null,
          "rendered": "Seminars for Parents",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "I have been providing seminars to parents and families for over 25 years. Parents often say, \\"Our kids didn't come with directions.\\" As a parent of 4 children and as a psychologist, I appreciate the challenges parents face in raising their children as healthy, happy and competent individuals. The goal of the seminars is to provide parents \\"directions\\" in understanding why their child is struggling and how they can help their child develop his/her body, brain, and emotions.",
          },
          "ref": null,
          "rendered": "I have been providing seminars to parents and families for over 25 years. Parents often say, \\"Our kids didn't come with directions.\\" As a parent of 4 children and as a psychologist, I appreciate the challenges parents face in raising their children as healthy, happy and competent individuals. The goal of the seminars is to provide parents \\"directions\\" in understanding why their child is struggling and how they can help their child develop his/her body, brain, and emotions.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "One of the most important skills children need to learn is what they can do when things become difficult in their life. Children can learn maladaptive coping strategies that can lead to disruption in the home, school and with friends. The focus of the workshops is to provide parents with resources to help their children develop skills in managing challenges in the different parts of their life. My approach is to provide practical and easy to understand strategies that can use on a daily basis.",
          },
          "ref": null,
          "rendered": "One of the most important skills children need to learn is what they can do when things become difficult in their life. Children can learn maladaptive coping strategies that can lead to disruption in the home, school and with friends. The focus of the workshops is to provide parents with resources to help their children develop skills in managing challenges in the different parts of their life. My approach is to provide practical and easy to understand strategies that can use on a daily basis.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Effective Parenting Seminars",
          },
          "ref": null,
          "rendered": "Effective Parenting Seminars",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "These are intensive parenting seminars that are offered several times a year. Each seminar is four-weeks. The seminar focuses on assisting parents in developing additional parenting skills. Topics include: school related issues, sibling issues, limit setting and emotional regulation. Specific plans are developed and time is given for parents to implement them. Parents will be able to discuss each week the effectiveness of the plans. The seminars are very practical and skill-based.",
          },
          "ref": null,
          "rendered": "These are intensive parenting seminars that are offered several times a year. Each seminar is four-weeks. The seminar focuses on assisting parents in developing additional parenting skills. Topics include: school related issues, sibling issues, limit setting and emotional regulation. Specific plans are developed and time is given for parents to implement them. Parents will be able to discuss each week the effectiveness of the plans. The seminars are very practical and skill-based.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Upcoming Seminars",
          },
          "ref": null,
          "rendered": "Upcoming Seminars",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "No upcoming seminars for parents. Please check back later.",
          },
          "ref": null,
          "rendered": "No upcoming seminars for parents. Please check back later.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
